src/app/components/hmcts-global-header/hmcts-global-header.component.ts

```diff
@@ -3,8 +3,10 @@ import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { Store } from '@ngrx/store';
 import { BehaviorSubject, Observable, of } from 'rxjs';
 import { map, switchMap, tap } from 'rxjs/operators';
+import { SearchStatePersistenceKey } from 'src/search/enums';
```

Comment:
@johnbenjamin-hmcts - please change to a relative import.

Reply:
Thank you @Daniel-Lam, my vs code auto-import is causing problems. I missed to track this one.